bindings/pyroot/pythonizations/test/pythonizedDefineAndFilters/Tests/readme.md

```diff
@@ -0,0 +1,17 @@
+# Tests for PyFilter
```

Comment:
This file should no longer exist once you reorganize the tests as suggested above. If you want to keep any of these comments, please place them in the docstring of the corresponding test class.